cli/pipeline/pipeline_definition.go

```diff
@@ -61,3 +61,15 @@ func (p *PipelineDefinitionStep) OverrideTag(tag string) {
 		p.Image = strings.Join(parts, ":")
 	}
 }
+
+func (p *PipelineDefinitionStep) OverrideVersion(version string) {
```

Comment:
`Version` and `Branch` are public, so we could just set them from outside. I'm not a Go expert so I don't know what the approach towards setters generally is, I guess this works!